tests/cli/set/test_family.py

```diff
@@ -10,7 +10,7 @@
 def test_set_family_without_options(cli_runner, base_context, base_store: Store, helpers):
     """Test to set a family using only the required arguments"""
     # GIVEN a database with a family
-    family_id = helpers.add_family(base_store).internal_id
+    family_id = helpers.add_case(base_store).internal_id
```

Comment:
```suggestion
    case_id = helpers.add_case(base_store).internal_id
```
Also for the rest of the module. And perhaps rename the `test_family.py` to `test_case.py`

Reply:
fixed